_wallscreens/silicon-valley/menuez_slideshow_adobe.html

```diff
@@ -0,0 +1,37 @@
+---
+title: "Douglas Menuez photography: Adobe"
+description: ""
+wallscreen: silicon-valley
+section: slideshow
+---
+<header class="title-area">
+  <h1>Silicon Valley Archives</h1>
+</header>
+
+<main class="content-area">
+  <pre>content-area</pre>
+</main>
+
+<aside class="card-area">
+  <div class="card-content">
+    <h2 class="experience-title">Douglas Menuez photography collection</h2>
+
+    <h3 class="item-title">Adobe</h3>
+    <p class="item-note">Adobe founder Chuck Geschke (left), Apple co-founder and NeXT founder Steve Jobs (center) and Adoube founder John Warnock.</p>
+    <div class="item-attribution"><span class="item-date">July 1990</span><span class="separator">•</span><span class="item-creator">Douglas Menuez</span></div>
+    <div class="item-collection"><i>From</i> Douglas Menuez photography collection, 1986 - 2006</div>
+
+    <p class="experience-summary">Douglas Menuez is a professional photographer who has documented progress in Silicon Valley during the period of approximately the early 1980s through the Internet boom, covering companies such as Adobe, Apple, NeXT, Intel, Sun Microsystems, IBM, venture capital deals, startups and Internet companies.</p>
+
+    <a href="">View catalog record</a>
+  </div>
+
+  <nav class="card-nav">
+    <a href="{% link _wallscreens/silicon-valley/menuez_slideshow_index.html %}" class="button">Previous</a>
+    <a href="" class="button disabled">Next</a>
```

Comment:
I see what you're doing, but the `cursor: not-allowed` isn't quite working here. I see the `not-allowed` icon ![](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor/not-allowed.gif) but I'm still able to click the link, which causes a page refresh. I'll stub a ticket out for when we revisit buttons.

Reply:
Oh, yeah. I didn't care about that because... they won't be disabled whenever we actually implement the content. We could drop `pointer-events: none;` on them, but 🤷